app/connectors/CalculatorConnector.scala

```diff
@@ -42,8 +40,8 @@ class CalculatorConnector @Inject()(http: DefaultHttpClient,
   implicit val hc: HeaderCarrier = HeaderCarrier().withExtraHeaders("Accept" -> "application/vnd.hmrc.1.0+json")
 
   def getMinimumDate()(implicit hc: HeaderCarrier): Future[LocalDate] = {
-    http.GET[DateTime](s"$serviceUrl/capital-gains-calculator/minimum-date").map { date =>
-      LocalDate.of(date.getYear, date.getMonthOfYear, date.getDayOfMonth)
+    http.GET[LocalDate](s"$serviceUrl/capital-gains-calculator/minimum-date").map { date =>
```

Comment:
The map can be dropped completeely as the GET now returns a LocalDate on its own.